open-source-trumps-innersource.md

```diff
@@ -0,0 +1,35 @@
+## Title
+Open Source trumps InnerSource
+
+## Statement of Problem
+* People find the InnerSource project but, after all things are considered, even if the InnerSource component meets their needs, they still go with the open source component.
+
+## Context
+* People look for software through external search engines.
+* Different Business Lines (BLs) are selecting different SW components to do the same function, resulting in inconsistencies in the user experience.
+
+## Forces 
+* There is a perception that the open source components are higher quality and more reusable. The channels to get any needed changes are more obvious with open source than with internal components. Can easily grab the open source whereas internally developed component has less visibility and ease of availability. (This could be separate Pattern). All things equal, they gravitate towards open source.
+* Open source should be voluntary; mandating people to use internally developed software goes against open source principles.
+* Internally developed components should have advantages over competing open source. If you can clearly demonstrate this, it can be persuasive.
+* Part of the reason this is a problem: if different teams are picking different open source components, the situation makes interoperability and integration more difficult; and this could potentially impact the user experience (inconsistency in behaviors).
+* Depending on the regulations within a company, it might be easier to use a corporate licensed, InnerSource component than an open source component which is licensed in a way that requires the company to disclose IP.
+* There may be many silos in the company; it would then be difficult to reach the developer base across those silos.
```

Comment:
It's unclear to me what these two last bullet points are trying to get at or how they are uniquely appropriate to describing this pattern?

Reply:
I think the point of this force is that it can be difficult to fight perception among dispersed and silo'd engineering staff and to communicate (1) that the inner source project exits, (2) that it has advantages over competing open source, (3) that there are legitimate reasons for preferring the inner source over competing open source software.